templates/minimal/README.md

```diff
@@ -37,6 +37,19 @@ A Polkadot SDK based project such as this one consists of:
 * 🛠️ Depending on your operating system and Rust version, there might be additional
 packages required to compile this template - please take note of the Rust compiler output.
 
+## Customization
+
+In case you would like to change the pallet's name from  `pallet-minimal-template` to `pallet-test`, you would need to implement the following changes:
```

Comment:
```suggestion
In case you would like to change the pallet's name from  `pallet-minimal-template` to `pallet-test`, 
you would need to implement the following changes:
```